pilot/pkg/networking/core/tracing.go

```diff
@@ -200,14 +195,6 @@ func configureFromProviderConfig(pushCtx *model.PushContext, proxy *model.Proxy,
 			}
 			return otelLightStepConfig(clusterName, hostname, provider.Lightstep.GetAccessToken())
 		}
-	case *meshconfig.MeshConfig_ExtensionProvider_Opencensus:
```

Comment:
we should probably add some form of warnings if these are used. 

Maybe in manifests/charts/istio-control/istio-discovery/templates/NOTES.txt for helm and operator/pkg/apis/istio/v1alpha1/validation/validation.go for operator?

can be a followup

Reply:
Ack.

Reply:
a warn log is adoptable.